scripts/TofConverter/converterGUI.py

```diff
@@ -136,6 +137,9 @@ def convert(self):
         except ArithmeticError as ae:
```

Comment:
Just for reference we could write this as using a tuple to catch multiple types:
```python
except (ArithmeticError, ValueError, RuntimeError) as exc:
    QMessageBox.warning(self, "TofConverter", str(exc))
    return

```

However, as we are doing the same thing each time would we be better just simplifying this to a single:
```python
except Exception as exc:
    QMessageBox.warning(self, "TofConverter", str(exc))
    return
```
as we really want to catch any error here?

Reply:
I'm hesitant to use `except Exception`, but the tuple version would be much neater.

Reply:
Could you help me understand the hesitation with the `Exception` catching? If we see a different exception type thrown in future then we're likely to just add it to this list as it's the case where we actually don't want any exceptions to escape this do we?